Misc/NEWS

```diff
@@ -303,6 +303,10 @@ Extension Modules
 Library
 -------
 
+- bpo-30014: selectors.EpollSelector.modify, selectors.PollSelector.modify
+  and selectors.DevpollSelector.modify are around 1.5x faster.
+  (Contributed by Giampaolo Rodola')
```

Comment:
Contributed by -> Patch by (but you may skip this and only attribute your name in the whatsnew document since you're already a core developer :))